docs/working-with-js-objects-and-values.md

```diff
@@ -1076,20 +1075,20 @@ not prevent consumers of the external function from calling it using values that
 are unsupported by the JavaScript function. Let’s see how we can use polymorphic
 variants to avoid runtime errors.
 
-If the values are strings, we can use the `mel.string` attribute:
+If the values are strings, we can directly use polymorphic variants:
```

Comment:
I changed this because Melange compiler now generates warning if you try to use `mel.string` in this way